
<!-- README.md is generated from README.Rmd. Please edit that file -->

# The dynamic landscape of peptide function prediction

This curated list collects peer-reviewed machine learning models
dedicated to predicting specific properties of peptides. For an
interactive and browse-able list, please visit:
<https://biogenies.info/peptide-prediction-list/>.

-   [Antiangiogenic peptides](#antiangiogenic-peptides)
-   [Antibacterial peptides (ABP)](#antibacterial-peptides-abp)
-   [Antibiofilm peptides](#antibiofilm-peptides)
-   [Anticancer peptides (ACP)](#anticancer-peptides-acp)
-   [Antifungal peptides (AFP)](#antifungal-peptides-afp)
-   [Antihypertensive peptides](#antihypertensive-peptides)
-   [Antiinflammatory peptides (AIP)](#antiinflammatory-peptides-aip)
-   [Antimicrobial peptides (AMP)](#antimicrobial-peptides-amp)
-   [Antiparasitic peptides (APP)](#antiparasitic-peptides-app)
-   [Antiviral peptides (AVP)](#antiviral-peptides-avp)
-   [Blood-brain barrier peptides
    (BBP)](#blood-brain-barrier-peptides-bbp)
-   [Cell-penetrating peptides (CPP)](#cell-penetrating-peptides-cpp)
-   [Chemotactic peptides](#chemotactic-peptides-)
-   [Neuropeptides peptides](#neuropeptides-peptides)
-   [Quorum sensing peptides](#quorum-sensing-peptides)
-   [Surface-binding peptides](#surface-binding-peptides)
-   [Toxic peptides (TP)](#toxic-peptides-tp)

## Antiangiogenic peptides

1.  AntAngioCOOL (2019):
    [publication](https://doi.org/10.1186/s12967-019-1813-7); [model
    repository](https://cran.r-project.org/web/packages/AntAngioCOOL/index.html).
2.  AntiAngioPred (2015):
    [publication](https://doi.org/10.1371/journal.pone.0136990); [web
    server](http://clri.res.in/subramanian/tools/antiangiopred/index.html).
3.  Pep-CNN (2022):
    [publication](https://doi.org/10.1016/j.chemolab.2022.104490);
    [training repository](https://github.com/alivelxj/Pep-CNN); [model
    repository](https://github.com/alivelxj/Pep-CNN).
4.  PEPred-Suite (2019):
    [publication](https://doi.org/10.1093/bioinformatics/btz246); [web
    server](http://server.malab.cn/PEPred-Suite/).
5.  PPTPP (2020):
    [publication](https://doi.org/10.1093/bioinformatics/btaa275);
    [training repository](https://github.com/YPZ858/PPTPP); [model
    repository](https://github.com/YPZ858/PPTPP).
6.  TargetAntiAngio (2019):
    [publication](https://doi.org/10.3390/ijms20122950); [training
    repository](https://github.com/Shoombuatong/TargetAntiAngio); [model
    repository](https://github.com/Shoombuatong/TargetAntiAngio); [web
    server](http://codes.bio/targetantiangio/).
7.  TPpred-ATMV (2022):
    [publication](https://doi.org/10.1093/bioinformatics/btac200);
    [model repository](https://github.com/cokeyk/TPpred-ATMV).
8.  Zhang et al. (2018):
    [publication](https://doi.org/10.1038/s41598-018-32443-w).

## Antibacterial peptides (ABP)

1.  ClassAMP (2012):
    [publication](https://doi.org/10.1109/tcbb.2012.89); [web
    server](http://www.bicnirrh.res.in/classamp/predict.php).
2.  Deep-ABPpred (2021):
    [publication](https://doi.org/10.1093/bib/bbab065); [web
    server](https://abppred.anvil.app/).
3.  iAMP-CA2L (2021):
    [publication](https://doi.org/10.1093/bib/bbab209); [training
    repository](https://github.com/liujin66/iAMP-CA2L); [model
    repository](https://github.com/liujin66/iAMP-CA2L); [web
    server](http://www.jci-bioinfo.cn/iAMP-CA2L).
4.  iAMPpred (2017): [publication](https://doi.org/10.1038/srep42362);
    [web server](http://cabgrid.res.in:8080/amppred).
5.  MAMPs-Pred (2019):
    [publication](https://doi.org/10.1186/s12859-019-2766-9).
6.  Pep-CNN (2022):
    [publication](https://doi.org/10.1016/j.chemolab.2022.104490);
    [training repository](https://github.com/alivelxj/Pep-CNN); [model
    repository](https://github.com/alivelxj/Pep-CNN).
7.  PEPred-Suite (2019):
    [publication](https://doi.org/10.1093/bioinformatics/btz246); [web
    server](http://server.malab.cn/PEPred-Suite/).
8.  PPTPP (2020):
    [publication](https://doi.org/10.1093/bioinformatics/btaa275);
    [training repository](https://github.com/YPZ858/PPTPP); [model
    repository](https://github.com/YPZ858/PPTPP).
9.  PreTP-EL (2021): [publication](https://doi.org/10.1093/bib/bbab358);
    [web server](http://bliulab.net/PreTP-EL/).
10. PreTP-Stack (2022):
    [publication](https://doi.org/10.1109/tcbb.2022.3183018); [web
    server](http://bliulab.net/PreTP-Stack/).
11. StaBle-ABPpred (2021):
    [publication](https://doi.org/10.1093/bib/bbab439); [web
    server](https://stable-abppred.anvil.app).
12. TP-MV (2022):
    [publication](https://doi.org/10.2174/1574893617666211220153429).
13. TPpred-ATMV (2022):
    [publication](https://doi.org/10.1093/bioinformatics/btac200);
    [model repository](https://github.com/cokeyk/TPpred-ATMV).
14. Zhang et al. (2021):
    [publication](https://doi.org/10.1093/bib/bbab200); [training
    repository](https://github.com/BioSequenceAnalysis/Bert-Protein);
    [model
    repository](https://github.com/BioSequenceAnalysis/Bert-Protein).

## Antibiofilm peptides

1.  dPABBs (2016): [publication](https://doi.org/10.1038/srep21839);
    [web
    server](https://ab-openlab.csir.res.in/abp/antibiofilm/index.php).
2.  iAMP-CA2L (2021):
    [publication](https://doi.org/10.1093/bib/bbab209); [training
    repository](https://github.com/liujin66/iAMP-CA2L); [model
    repository](https://github.com/liujin66/iAMP-CA2L); [web
    server](http://www.jci-bioinfo.cn/iAMP-CA2L).

## Anticancer peptides (ACP)

1.  ACHP (2021):
    [publication](https://doi.org/10.1007/s10989-021-10222-y); [web
    server](http://118.178.58.31:9801/).
2.  ACP-2DCNN (2022):
    [publication](https://doi.org/10.1016/j.chemolab.2022.104589).
3.  ACP-DA (2021):
    [publication](https://doi.org/10.3389/fgene.2021.698477); [model
    repository](https://github.com/chenxgscuec/ACPDA).
4.  ACP-DL (2019):
    [publication](https://doi.org/10.1016/j.omtn.2019.04.025); [training
    repository](https://github.com/haichengyi/ACP-DL); [model
    repository](https://github.com/haichengyi/ACP-DL).
5.  ACP-MHCNN (2021):
    [publication](https://doi.org/10.1038/s41598-021-02703-3); [training
    repository](https://github.com/mrzResearchArena/Anticancer-Peptides-CNN);
    [model
    repository](https://github.com/mrzResearchArena/Anticancer-Peptides-CNN);
    [web server](https://anticancer.pythonanywhere.com/).
6.  ACPNet (2022):
    [publication](https://doi.org/10.3390/molecules27051544); [training
    repository](https://github.com/abcair/ACPNet); [model
    repository](https://github.com/abcair/ACPNet).
7.  ACPP (2014):
    [publication](https://doi.org/10.1007/s10989-014-9435-7); [web
    server](http://acpp.bicpu.edu.in/predict.php).
8.  ACPred (2019):
    [publication](https://doi.org/10.3390/molecules24101973); [web
    server](http://codes.bio/acpred).
9.  ACPred-FL (2018):
    [publication](https://doi.org/10.1093/bioinformatics/bty451); [web
    server](http://server.malab.cn/ACPred-FL).
10. ACPred-Fuse (2019):
    [publication](https://doi.org/10.1093/bib/bbz088); [web
    server](http://server.malab.cn/ACPred-Fuse).
11. AntiCP (2013): [publication](https://doi.org/10.1038/srep02984);
    [web server](http://crdd.osdd.net/raghava/anticp).
12. AntiCP 2.0 (2020):
    [publication](https://doi.org/10.1093/bib/bbaa153); [model
    repository](https://github.com/raghavagps/anticp2); [web
    server](https://webs.iiitd.edu.in/raghava/anticp2).
13. cACP-2LFS (2020):
    [publication](https://doi.org/10.1109/access.2020.3009125);
    [training repository](https://github.com/shahidawkum/cACP-2LFS);
    [model repository](https://github.com/shahidawkum/cACP-2LFS).
14. CancerGram (2020):
    [publication](https://doi.org/10.3390/pharmaceutics12111045);
    [training
    repository](https://github.com/BioGenies/CancerGram-analysis);
    [model repository](https://github.com/BioGenies/CancerGram); [web
    server](http://biongram.biotech.uni.wroc.pl/CancerGram).
15. DeepACP (2020):
    [publication](https://doi.org/10.1016/j.omtn.2020.10.005); [training
    repository](https://github.com/jingry/autoBioSeqpy/tree/master/examples/anticancer_peptide_prediction);
    [model
    repository](https://github.com/jingry/autoBioSeqpy/tree/master/examples/anticancer_peptide_prediction).
16. DLACP (2020):
    [publication](https://doi.org/10.1109/bibm49941.2020.9313572).
17. DLFF-ACP (2021): [publication](https://doi.org/10.7717/peerj.11906);
    [training repository](https://github.com/wame-ng/DLFF-ACP); [model
    repository](https://github.com/wame-ng/DLFF-ACP).
18. DRACP (2020):
    [publication](https://doi.org/10.1186/s12859-020-03812-y).
19. Hajisharifi et al. (2014):
    [publication](https://doi.org/10.1016/j.jtbi.2013.08.037).
20. iACP (2016):
    [publication](https://doi.org/10.18632/oncotarget.7815); [web
    server](http://lin-group.cn/server/iACP).
21. iACP-DRLF (2021):
    [publication](https://doi.org/10.1093/bib/bbab008); [model
    repository](https://github.com/zhibinlv/iACP-DRLF).
22. iACP-FSCM (2021):
    [publication](https://doi.org/10.1038/s41598-021-82513-9); [training
    repository](https://github.com/Shoombuatong/Dataset-Code/tree/master/iACP-FSCM);
    [model
    repository](https://github.com/Shoombuatong/Dataset-Code/tree/master/iACP-FSCM);
    [web server](http://camt.pythonanywhere.com/iACP-FSCM).
23. iACP-GAEnsC (2017):
    [publication](https://doi.org/10.1016/j.artmed.2017.06.008).
24. iAMP-CA2L (2021):
    [publication](https://doi.org/10.1093/bib/bbab209); [training
    repository](https://github.com/liujin66/iAMP-CA2L); [model
    repository](https://github.com/liujin66/iAMP-CA2L); [web
    server](http://www.jci-bioinfo.cn/iAMP-CA2L).
25. Khan et al. (2017):
    [publication](https://doi.org/10.1145/3168776.3168787).
26. Li & Wang (2016): [publication](https://doi.org/10.1038/srep33910).
27. Li et al. (2020):
    [publication](https://doi.org/10.3389/fbioe.2020.00892).
28. mACPpred (2019):
    [publication](https://doi.org/10.3390/ijms20081964); [web
    server](http://www.thegleelab.org/mACPpred).
29. MAMPs-Pred (2019):
    [publication](https://doi.org/10.1186/s12859-019-2766-9).
30. ME-ACP (2022):
    [publication](https://doi.org/10.1016/j.compbiomed.2022.105459);
    [training repository](https://github.com/XDhyao/ME-ACP); [model
    repository](https://github.com/XDhyao/ME-ACP).
31. MLACP (2017):
    [publication](https://doi.org/10.18632/oncotarget.20365); [web
    server](http://www.thegleelab.org/MLACP/MLACP.html).
32. MLBP (2021): [publication](https://doi.org/10.1093/bib/bbab414);
    [training repository](https://github.com/xialab-ahu/MLBP); [model
    repository](https://github.com/xialab-ahu/MLBP); [web
    server](http://bioinfo.ahu.edu.cn/MLBP/).
33. MPMABP (2022): [publication](https://doi.org/10.3390/ph15060707);
    [training
    repository](https://drive.google.com/drive/folders/10oPJ_koydNV4Cw_BjpwLtiLq7CpaNlGN);
    [model repository](https://github.com/Good-Ly/MPMABP).
34. Pep-CNN (2022):
    [publication](https://doi.org/10.1016/j.chemolab.2022.104490);
    [training repository](https://github.com/alivelxj/Pep-CNN); [model
    repository](https://github.com/alivelxj/Pep-CNN).
35. PPTPP (2020):
    [publication](https://doi.org/10.1093/bioinformatics/btaa275);
    [training repository](https://github.com/YPZ858/PPTPP); [model
    repository](https://github.com/YPZ858/PPTPP).
36. PreTP-EL (2021): [publication](https://doi.org/10.1093/bib/bbab358);
    [web server](http://bliulab.net/PreTP-EL/).
37. PreTP-Stack (2022):
    [publication](https://doi.org/10.1109/tcbb.2022.3183018); [web
    server](http://bliulab.net/PreTP-Stack/).
38. SAP (2018): [publication](https://doi.org/10.3390/genes9030158).
39. StackACPred (2022):
    [publication](https://doi.org/10.1016/j.chemolab.2021.104458);
    [training
    repository](https://github.com/Muhammad-Arif-NUST/StackACPred-);
    [model
    repository](https://github.com/Muhammad-Arif-NUST/StackACPred-).
40. TargetACP (2018):
    [publication](https://doi.org/10.1016/j.chemolab.2018.09.007).
41. TPpred-ATMV (2022):
    [publication](https://doi.org/10.1093/bioinformatics/btac200);
    [model repository](https://github.com/cokeyk/TPpred-ATMV).
42. xDeep-AcPEP (2021):
    [publication](https://doi.org/10.1021/acs.jcim.1c00181); [training
    repository](https://drive.google.com/drive/folders/1DXHppIKO0vNqvpGrFAQyqnBodi3dr3fX);
    [model repository](https://github.com/chen709847237/xDeep-AcPEP);
    [web server](https://app.cbbio.online/acpep/home).

## Antifungal peptides (AFP)

1.  Antifp (2018):
    [publication](https://doi.org/10.3389/fmicb.2018.00323); [web
    server](https://webs.iiitd.edu.in/raghava/antifp).
2.  ClassAMP (2012):
    [publication](https://doi.org/10.1109/tcbb.2012.89); [web
    server](http://www.bicnirrh.res.in/classamp/predict.php).
3.  Deep-AFPpred (2021):
    [publication](https://doi.org/10.1093/bib/bbab422); [web
    server](https://afppred.anvil.app/).
4.  Deep-AntiFP (2021):
    [publication](https://doi.org/10.1016/j.chemolab.2020.104214);
    [model repository](https://github.com/shahidawkum/Deep-AntiFP).
5.  iAFPs-EnC-GA (2022):
    [publication](https://doi.org/10.1016/j.chemolab.2022.104516);
    [model repository](https://github.com/farmanit335/iAFPs-EnC-GA).
6.  iAMP-CA2L (2021):
    [publication](https://doi.org/10.1093/bib/bbab209); [training
    repository](https://github.com/liujin66/iAMP-CA2L); [model
    repository](https://github.com/liujin66/iAMP-CA2L); [web
    server](http://www.jci-bioinfo.cn/iAMP-CA2L).
7.  iAMPpred (2017): [publication](https://doi.org/10.1038/srep42362);
    [web server](http://cabgrid.res.in:8080/amppred).
8.  Mousavizadegan & Mohabatkar (2018):
    [publication](https://doi.org/10.1142/s0219720018500166).
9.  PEPred-Suite (2019):
    [publication](https://doi.org/10.1093/bioinformatics/btz246); [web
    server](http://server.malab.cn/PEPred-Suite/).
10. TCN-AFPpred (2022):
    [publication](https://doi.org/10.1093/bib/bbac008); [web
    server](https://tcn-afppred.anvil.app/).

## Antihypertensive peptides

1.  ACHP (2021):
    [publication](https://doi.org/10.1007/s10989-021-10222-y); [web
    server](http://118.178.58.31:9801/).
2.  mAHTPred (2018):
    [publication](https://doi.org/10.1093/bioinformatics/bty1047); [web
    server](http://thegleelab.org/mAHTPred/index.html).
3.  MLBP (2021): [publication](https://doi.org/10.1093/bib/bbab414);
    [training repository](https://github.com/xialab-ahu/MLBP); [model
    repository](https://github.com/xialab-ahu/MLBP); [web
    server](http://bioinfo.ahu.edu.cn/MLBP/).
4.  MPMABP (2022): [publication](https://doi.org/10.3390/ph15060707);
    [training
    repository](https://drive.google.com/drive/folders/10oPJ_koydNV4Cw_BjpwLtiLq7CpaNlGN);
    [model repository](https://github.com/Good-Ly/MPMABP).
5.  PAAP (2018): [publication](https://doi.org/10.4155/fmc-2017-0300);
    [training repository](https://github.com/chaninlab/PAAP); [model
    repository](https://github.com/chaninlab/PAAP); [web
    server](https://codes.bio/paap/).
6.  Rauf et al. (2021):
    [publication](https://doi.org/10.3390/app11052316).
7.  Wang et al. (2021):
    [publication](https://doi.org/10.3390/foods10030550).

## Antiinflammatory peptides (AIP)

1.  AIEpred (2021):
    [publication](https://doi.org/10.1109/tcbb.2020.2968419); [training
    repository](https://github.com/guofei-tju/Ensemble-classifier-chain-model);
    [model
    repository](https://github.com/guofei-tju/Ensemble-classifier-chain-model).
2.  AIPpred (2018):
    [publication](https://doi.org/10.3389/fphar.2018.00276); [web
    server](http://www.thegleelab.org/AIPpred).
3.  AntiFlamPred (2021):
    [publication](https://doi.org/10.32604/cmc.2021.017297).
4.  AntiInflam (2017):
    [publication](https://doi.org/10.1186/s12967-016-1103-6); [web
    server](http://metagenomics.iiserb.ac.in/antiinflam).
5.  MLBP (2021): [publication](https://doi.org/10.1093/bib/bbab414);
    [training repository](https://github.com/xialab-ahu/MLBP); [model
    repository](https://github.com/xialab-ahu/MLBP); [web
    server](http://bioinfo.ahu.edu.cn/MLBP/).
6.  MPMABP (2022): [publication](https://doi.org/10.3390/ph15060707);
    [training
    repository](https://drive.google.com/drive/folders/10oPJ_koydNV4Cw_BjpwLtiLq7CpaNlGN);
    [model repository](https://github.com/Good-Ly/MPMABP).
7.  Pep-CNN (2022):
    [publication](https://doi.org/10.1016/j.chemolab.2022.104490);
    [training repository](https://github.com/alivelxj/Pep-CNN); [model
    repository](https://github.com/alivelxj/Pep-CNN).
8.  PEPred-Suite (2019):
    [publication](https://doi.org/10.1093/bioinformatics/btz246); [web
    server](http://server.malab.cn/PEPred-Suite/).
9.  PPTPP (2020):
    [publication](https://doi.org/10.1093/bioinformatics/btaa275);
    [training repository](https://github.com/YPZ858/PPTPP); [model
    repository](https://github.com/YPZ858/PPTPP).
10. PreAIP (2019):
    [publication](https://doi.org/10.3389/fgene.2019.00129); [web
    server](http://kurata14.bio.kyutech.ac.jp/PreAIP).
11. PreTP-EL (2021): [publication](https://doi.org/10.1093/bib/bbab358);
    [web server](http://bliulab.net/PreTP-EL/).
12. PreTP-Stack (2022):
    [publication](https://doi.org/10.1109/tcbb.2022.3183018); [web
    server](http://bliulab.net/PreTP-Stack/).
13. TPpred-ATMV (2022):
    [publication](https://doi.org/10.1093/bioinformatics/btac200);
    [model repository](https://github.com/cokeyk/TPpred-ATMV).

## Antimicrobial peptides (AMP)

1.  ACEP (2020):
    [publication](https://doi.org/10.1186/s12864-020-06978-0); [training
    repository](https://github.com/Fuhaoyi/ACEP); [model
    repository](https://github.com/Fuhaoyi/ACEP).
2.  AI4AMP (2021):
    [publication](https://doi.org/10.1128/msystems.00299-21); [training
    repository](https://axp.iis.sinica.edu.tw/AI4AMP/helppage.html);
    [model repository](https://github.com/LinTzuTang/AI4AMP_predictor);
    [web server](http://symbiosis.iis.sinica.edu.tw/PC_6/).
3.  AMAP (2019):
    [publication](https://doi.org/10.1016/j.compbiomed.2019.02.018);
    [web server](http://amap.pythonanywhere.com/).
4.  AMP0 (2022):
    [publication](https://doi.org/10.1109/tcbb.2020.2999399); [web
    server](http://ampzero.pythonanywhere.com).
5.  AMPDiscover (2021):
    [publication](https://doi.org/10.1021/acs.jcim.1c00251); [web
    server](https://biocom-ampdiscover.cicese.mx/).
6.  AmPEP (2018):
    [publication](https://doi.org/10.1038/s41598-018-19752-w); [model
    repository](https://github.com/siuwengin/AmPEP); [web
    server](https://app.cbbio.online/ampep/home).
7.  amPEPpy 1.0 (2020):
    [publication](https://doi.org/10.1093/bioinformatics/btaa917);
    [training repository](https://github.com/tlawrence3/amPEPpy); [model
    repository](https://github.com/tlawrence3/amPEPpy).
8.  AmpGram (2020): [publication](https://doi.org/10.3390/ijms21124310);
    [training repository](https://github.com/michbur/AmpGram-analysis);
    [model repository](https://github.com/michbur/AmpGram); [web
    server](http://biongram.biotech.uni.wroc.pl/AmpGram/).
9.  Ampir (2020):
    [publication](https://doi.org/10.1093/bioinformatics/btaa653);
    [training repository](https://github.com/Legana/AMP_pub); [model
    repository](https://github.com/legana/amp_pub); [web
    server](https://ampir.marine-omics.net/).
10. AMPlify (2022):
    [publication](https://doi.org/10.1186/s12864-022-08310-4); [training
    repository](https://github.com/bcgsc/AMPlify); [model
    repository](https://github.com/bcgsc/AMPlify).
11. AMPpred-EL (2022):
    [publication](https://doi.org/10.1016/j.compbiomed.2022.105577).
12. AMPScanner V2 (2018):
    [publication](https://doi.org/10.1093/bioinformatics/bty179); [web
    server](https://www.dveltri.com/ascan/v2/ascan.html).
13. ANFIS (2012): [publication](https://doi.org/10.1002/bip.22066).
14. AniAMPpred (2021):
    [publication](https://doi.org/10.1093/bib/bbab242); [web
    server](https://aniamppred.anvil.app/).
15. APIN (2019):
    [publication](https://doi.org/10.1186/s12859-019-3327-y); [training
    repository](https://github.com/zhanglabNKU/APIN); [model
    repository](https://github.com/zhanglabNKU/APIN).
16. CAMP (2009): [publication](https://doi.org/10.1093/nar/gkp1021).
17. CAMPR3 (2015): [publication](https://doi.org/10.1093/nar/gkv1051);
    [web server](http://www.camp.bicnirrh.res.in/predict/).
18. ClassAMP (2012):
    [publication](https://doi.org/10.1109/tcbb.2012.89); [web
    server](http://www.bicnirrh.res.in/classamp/predict.php).
19. Co-AMPpred (2021):
    [publication](https://doi.org/10.1186/s12859-021-04305-2); [training
    repository](https://github.com/onkarS23/CoAMPpred); [model
    repository](https://github.com/onkarS23/CoAMPpred).
20. CS-AMPPred (2012):
    [publication](https://doi.org/10.1371/journal.pone.0051444); [model
    repository](https://sourceforge.net/projects/csamppred/).
21. DBAASP (2014):
    [publication](https://doi.org/10.1111/1574-6968.12489); [web
    server](https://dbaasp.org/prediction/general).
22. Deep-AmPEP30 (2020):
    [publication](https://doi.org/10.1016/j.omtn.2020.05.006); [training
    repository](https://sourceforge.net/projects/axpep/); [model
    repository](https://sourceforge.net/projects/axpep/); [web
    server](https://app.cbbio.online/ampep/home).
23. Ensemble-AMPPred (2021):
    [publication](https://doi.org/10.3390/genes12020137); [training
    repository](http://ncrna-pred.com/Hybrid_AMPPred.htm); [model
    repository](http://ncrna-pred.com/Hybrid_AMPPred.htm).
24. Healy et al. (2021):
    [publication](https://doi.org/10.1007/978-3-030-86258-9_2); [model
    repository](https://github.com/gmit-amp-res/amp).
25. iAMP-2L (2013):
    [publication](https://doi.org/10.1016/j.ab.2013.01.019); [web
    server](http://www.jci-bioinfo.cn/iAMP-2L).
26. iAMP-CA2L (2021):
    [publication](https://doi.org/10.1093/bib/bbab209); [training
    repository](https://github.com/liujin66/iAMP-CA2L); [model
    repository](https://github.com/liujin66/iAMP-CA2L); [web
    server](http://www.jci-bioinfo.cn/iAMP-CA2L).
27. IAMPE (2020):
    [publication](https://doi.org/10.1021/acs.jcim.0c00841); [web
    server](http://cbb1.ut.ac.ir/).
28. Ma et al. (2022):
    [publication](https://doi.org/10.1038/s41587-022-01226-0); [model
    repository](https://github.com/mayuefine/c_AMPs-prediction).
29. Maccari et al. (2013):
    [publication](https://doi.org/10.1371/journal.pcbi.1003212).
30. Macrel (2020): [publication](https://doi.org/10.7717/peerj.10555);
    [training repository](https://github.com/BigDataBiology/macrel);
    [model repository](https://github.com/BigDataBiology/macrel); [web
    server](https://big-data-biology.org/software/macrel).
31. MAMPs-Pred (2019):
    [publication](https://doi.org/10.1186/s12859-019-2766-9).
32. MLAMP (2016):
    [publication](https://doi.org/10.1093/bioinformatics/btw560); [web
    server](http://www.jci-bioinfo.cn/MLAMP).
33. MLBP (2021): [publication](https://doi.org/10.1093/bib/bbab414);
    [training repository](https://github.com/xialab-ahu/MLBP); [model
    repository](https://github.com/xialab-ahu/MLBP); [web
    server](http://bioinfo.ahu.edu.cn/MLBP/).
34. MPMABP (2022): [publication](https://doi.org/10.3390/ph15060707);
    [training
    repository](https://drive.google.com/drive/folders/10oPJ_koydNV4Cw_BjpwLtiLq7CpaNlGN);
    [model repository](https://github.com/Good-Ly/MPMABP).
35. Ng et al. (2015):
    [publication](https://doi.org/10.1155/2015/212715).
36. PepVAE3 (2021):
    [publication](https://doi.org/10.3389/fmicb.2021.725727).
37. sAMP-PFPDeep (2021):
    [publication](https://doi.org/10.1093/bib/bbab487); [model
    repository](https://github.com/WaqarHusain/sAMP-PFPDeep).
38. Wang et al. (2011):
    [publication](https://doi.org/10.1371/journal.pone.0018476); [web
    server](http://amp.biosino.org/.).
39. Wani et al. (2021):
    [publication](https://doi.org/10.1007/s11517-021-02443-6); [model
    repository](https://static-content.springer.com/esm/art%3A10.1007%2Fs11517-021-02443-6/MediaObjects/11517_2021_2443_MOESM1_ESM.docx).
40. Zhang et al. (2021):
    [publication](https://doi.org/10.1093/bib/bbab200); [training
    repository](https://github.com/BioSequenceAnalysis/Bert-Protein);
    [model
    repository](https://github.com/BioSequenceAnalysis/Bert-Protein).

## Antiparasitic peptides (APP)

1.  i2APP (2022):
    [publication](https://doi.org/10.3389/fgene.2022.884589); [training
    repository](https://github.com/greyspring/i2APP); [model
    repository](https://github.com/greyspring/i2APP).
2.  MAMPs-Pred (2019):
    [publication](https://doi.org/10.1186/s12859-019-2766-9).
3.  PredAPP (2021):
    [publication](https://doi.org/10.1007/s12539-021-00484-x); [training
    repository](https://github.com/xialab-ahu/PredAPP); [model
    repository](https://github.com/xialab-ahu/PredAPP); [web
    server](http://predapp.xialab.info/RunClassifier).

## Antiviral peptides (AVP)

1.  AntiVPP 1.0 (2019):
    [publication](https://doi.org/10.1016/j.compbiomed.2019.02.011);
    [model repository](https://github.com/bio-coding/AntiVPP).
2.  AVP‐IC50Pred (2015):
    [publication](https://doi.org/10.1002/bip.22703); [web
    server](http://crdd.osdd.net/servers/ic50avp).
3.  AVPIden (2021): [publication](https://doi.org/10.1093/bib/bbab263);
    [model repository](https://github.com/BiOmicsLab/AVPIden); [web
    server](https://awi.cuhk.edu.cn/AVPIden/).
4.  AVPpred (2012): [publication](https://doi.org/10.1093/nar/gks450);
    [web server](http://crdd.osdd.net/servers/avppred/submit.php).
5.  Bakare et al. (2021):
    [publication](https://doi.org/10.1038/s41598-021-00223-8).
6.  ClassAMP (2012):
    [publication](https://doi.org/10.1109/tcbb.2012.89); [web
    server](http://www.bicnirrh.res.in/classamp/predict.php).
7.  DeepAVP (2020):
    [publication](https://doi.org/10.1109/jbhi.2020.2977091); [web
    server](http://www.lbci.cn/deepavp/index.html).
8.  ENNAVIA (2021): [publication](https://doi.org/10.1093/bib/bbab258);
    [web server](https://research.timmons.eu/ennavia).
9.  FIRM-AVP (2020):
    [publication](https://doi.org/10.1038/s41598-020-76161-8); [training
    repository](https://github.com/pmartR/FIRM-AVP); [model
    repository](https://github.com/pmartR/FIRM-AVP); [web
    server](https://msc-viz.emsl.pnnl.gov/AVPR/).
10. iACVP (2022): [publication](https://doi.org/10.1093/bib/bbac265);
    [training repository](https://github.com/kuratahiroyuki/iACVP);
    [model repository](https://github.com/kuratahiroyuki/iACVP); [web
    server](http://kurata35.bio.kyutech.ac.jp/iACVP/).
11. iAMPpred (2017): [publication](https://doi.org/10.1038/srep42362);
    [web server](http://cabgrid.res.in:8080/amppred).
12. MAMPs-Pred (2019):
    [publication](https://doi.org/10.1186/s12859-019-2766-9).
13. Meta-iAVP (2019):
    [publication](https://doi.org/10.3390/ijms20225743); [web
    server](http://codes.bio/meta-iavp).
14. Pep-CNN (2022):
    [publication](https://doi.org/10.1016/j.chemolab.2022.104490);
    [training repository](https://github.com/alivelxj/Pep-CNN); [model
    repository](https://github.com/alivelxj/Pep-CNN).
15. PEPred-Suite (2019):
    [publication](https://doi.org/10.1093/bioinformatics/btz246); [web
    server](http://server.malab.cn/PEPred-Suite/).
16. PPTPP (2020):
    [publication](https://doi.org/10.1093/bioinformatics/btaa275);
    [training repository](https://github.com/YPZ858/PPTPP); [model
    repository](https://github.com/YPZ858/PPTPP).
17. PreAntiCoV (2021):
    [publication](https://doi.org/10.1093/bib/bbaa423); [training
    repository](https://github.com/poncey/PreAntiCoV); [model
    repository](https://github.com/poncey/PreAntiCoV).
18. PreTP-EL (2021): [publication](https://doi.org/10.1093/bib/bbab358);
    [web server](http://bliulab.net/PreTP-EL/).
19. PreTP-Stack (2022):
    [publication](https://doi.org/10.1109/tcbb.2022.3183018); [web
    server](http://bliulab.net/PreTP-Stack/).
20. TPpred-ATMV (2022):
    [publication](https://doi.org/10.1093/bioinformatics/btac200);
    [model repository](https://github.com/cokeyk/TPpred-ATMV).

## Blood-brain barrier peptides (BBP)

1.  B3Pred (2021):
    [publication](https://doi.org/10.3390/pharmaceutics13081237); [web
    server](https://webs.iiitd.edu.in/raghava/b3pred/).
2.  BBPpred (2021):
    [publication](https://doi.org/10.1021/acs.jcim.0c01115); [web
    server](http://BBPpred.xialab.info).
3.  Zou (2021): [publication](https://doi.org/10.1002/pep2.24247).

## Cell-penetrating peptides (CPP)

1.  BChemRF-CPPred (2021):
    [publication](https://doi.org/10.1038/s41598-021-87134-w); [web
    server](http://comptools.linc.ufpa.br/BChemRF-CPPred).
2.  CPPred-RF (2017):
    [publication](https://doi.org/10.1021/acs.jproteome.7b00019); [web
    server](http://server.malab.cn/CPPred-RF/).
3.  ITP-Pred (2020): [publication](https://doi.org/10.1093/bib/bbaa367).
4.  KELM-CPPpred (2018):
    [publication](https://doi.org/10.1021/acs.jproteome.8b00322); [web
    server](http://sairam.people.iitgn.ac.in/KELM-CPPpred.html).
5.  Liu et al. (2022):
    [publication](https://doi.org/10.3390/app12115383).
6.  MLCPP (2018):
    [publication](https://doi.org/10.1021/acs.jproteome.8b00148); [web
    server](http://www.thegleelab.org/MLCPP).
7.  MLCPP 2.0 (2022):
    [publication](https://doi.org/10.1016/j.jmb.2022.167604); [web
    server](https://balalab-skku.org/mlcpp2).
8.  Pep-CNN (2022):
    [publication](https://doi.org/10.1016/j.chemolab.2022.104490);
    [training repository](https://github.com/alivelxj/Pep-CNN); [model
    repository](https://github.com/alivelxj/Pep-CNN).
9.  PEPred-Suite (2019):
    [publication](https://doi.org/10.1093/bioinformatics/btz246); [web
    server](http://server.malab.cn/PEPred-Suite/).
10. PPTPP (2020):
    [publication](https://doi.org/10.1093/bioinformatics/btaa275);
    [training repository](https://github.com/YPZ858/PPTPP); [model
    repository](https://github.com/YPZ858/PPTPP).
11. PreTP-EL (2021): [publication](https://doi.org/10.1093/bib/bbab358);
    [web server](http://bliulab.net/PreTP-EL/).
12. PreTP-Stack (2022):
    [publication](https://doi.org/10.1109/tcbb.2022.3183018); [web
    server](http://bliulab.net/PreTP-Stack/).
13. SkipCPP-Pred (2017):
    [publication](https://doi.org/10.1186/s12864-017-4128-1); [web
    server](http://server.malab.cn/SkipCPP-Pred/Index.html).
14. TargetCPP (2020):
    [publication](https://doi.org/10.1007/s10822-020-00307-z); [training
    repository](https://static-content.springer.com/esm/art%3A10.1007%2Fs10822-020-00307-z/MediaObjects/10822_2020_307_MOESM2_ESM.zip);
    [model
    repository](https://static-content.springer.com/esm/art%3A10.1007%2Fs10822-020-00307-z/MediaObjects/10822_2020_307_MOESM2_ESM.zip).
15. TP-MV (2022):
    [publication](https://doi.org/10.2174/1574893617666211220153429).
16. TPpred-ATMV (2022):
    [publication](https://doi.org/10.1093/bioinformatics/btac200);
    [model repository](https://github.com/cokeyk/TPpred-ATMV).

## Chemotactic peptides

1.  iAMP-CA2L (2021):
    [publication](https://doi.org/10.1093/bib/bbab209); [training
    repository](https://github.com/liujin66/iAMP-CA2L); [model
    repository](https://github.com/liujin66/iAMP-CA2L); [web
    server](http://www.jci-bioinfo.cn/iAMP-CA2L).

## Neuropeptides peptides

1.  NeuroPIpred (2019):
    [publication](https://doi.org/10.1038/s41598-019-41538-x); [web
    server](http://i.uestc.edu.cn/neuropeptide/neuropp/home.html).
2.  NeuroPP (2018):
    [publication](https://doi.org/10.1007/s12539-018-0287-2); [web
    server](http://i.uestc.edu.cn/neuropeptide/neuropp/home.html).
3.  NeuroPpred-Fuse (2021):
    [publication](https://doi.org/10.1093/bib/bbab310); [model
    repository](https://github.com/mingmingjiang1/NeuroPpred-Fuse).
4.  NeuroPred-FRL (2021):
    [publication](https://doi.org/10.1093/bib/bbab167); [web
    server](http://kurata14.bio.kyutech.ac.jp/NeuroPred-FRL/prediction.php).
5.  PredNeuroP (2020):
    [publication](https://doi.org/10.1021/acs.jproteome.0c00276);
    [training repository](https://github.com/xialab-ahu/PredNeuroP);
    [model repository](https://github.com/xialab-ahu/PredNeuroP).

## Quorum sensing peptides

1.  EnsembleQS (2022):
    [publication](https://doi.org/10.1007/s12275-022-2044-9); [training
    repository](https://github.com/proteinexplorers/EnsembleQS); [model
    repository](https://github.com/proteinexplorers/EnsembleQS).
2.  iQSP (2019): [publication](https://doi.org/10.3390/ijms21010075);
    [web server](http://codes.bio/iqsp/).
3.  ITP-Pred (2020): [publication](https://doi.org/10.1093/bib/bbaa367).
4.  Pep-CNN (2022):
    [publication](https://doi.org/10.1016/j.chemolab.2022.104490);
    [training repository](https://github.com/alivelxj/Pep-CNN); [model
    repository](https://github.com/alivelxj/Pep-CNN).
5.  PEPred-Suite (2019):
    [publication](https://doi.org/10.1093/bioinformatics/btz246); [web
    server](http://server.malab.cn/PEPred-Suite/).
6.  PPTPP (2020):
    [publication](https://doi.org/10.1093/bioinformatics/btaa275);
    [training repository](https://github.com/YPZ858/PPTPP); [model
    repository](https://github.com/YPZ858/PPTPP).
7.  PreTP-EL (2021): [publication](https://doi.org/10.1093/bib/bbab358);
    [web server](http://bliulab.net/PreTP-EL/).
8.  PreTP-Stack (2022):
    [publication](https://doi.org/10.1109/tcbb.2022.3183018); [web
    server](http://bliulab.net/PreTP-Stack/).
9.  QSPpred (2015):
    [publication](https://doi.org/10.1371/journal.pone.0120066); [web
    server](http://crdd.osdd.net/servers/qsppred/).
10. QSPred-FL (2018): [publication](https://doi.org/10.1093/bib/bby107);
    [web server](http://server.malab.cn/QSPred-FL).
11. TP-MV (2022):
    [publication](https://doi.org/10.2174/1574893617666211220153429).
12. TPpred-ATMV (2022):
    [publication](https://doi.org/10.1093/bioinformatics/btac200);
    [model repository](https://github.com/cokeyk/TPpred-ATMV).

## Surface-binding peptides

1.  Pep-CNN (2022):
    [publication](https://doi.org/10.1016/j.chemolab.2022.104490);
    [training repository](https://github.com/alivelxj/Pep-CNN); [model
    repository](https://github.com/alivelxj/Pep-CNN).
2.  PEPred-Suite (2019):
    [publication](https://doi.org/10.1093/bioinformatics/btz246); [web
    server](http://server.malab.cn/PEPred-Suite/).
3.  PPTPP (2020):
    [publication](https://doi.org/10.1093/bioinformatics/btaa275);
    [training repository](https://github.com/YPZ858/PPTPP); [model
    repository](https://github.com/YPZ858/PPTPP).
4.  PreTP-EL (2021): [publication](https://doi.org/10.1093/bib/bbab358);
    [web server](http://bliulab.net/PreTP-EL/).
5.  PreTP-Stack (2022):
    [publication](https://doi.org/10.1109/tcbb.2022.3183018); [web
    server](http://bliulab.net/PreTP-Stack/).
6.  TPpred-ATMV (2022):
    [publication](https://doi.org/10.1093/bioinformatics/btac200);
    [model repository](https://github.com/cokeyk/TPpred-ATMV).

## Toxic peptides (TP)

1.  ATSE (2021): [publication](https://doi.org/10.1093/bib/bbab041);
    [web server](https://server.malab.cn/ATSE).
2.  ToxIBTL (2022):
    [publication](https://doi.org/10.1093/bioinformatics/btac006);
    [model repository](https://github.com/WLYLab/ToxIBTL); [web
    server](https://server.wei-group.net/ToxIBTL/Server.html).
3.  ToxinPred (2014):
    [publication](https://doi.org/10.1007/978-1-4939-2285-7_7); [web
    server](https://webs.iiitd.edu.in/raghava/toxinpred/algo.php).

### About and citation

This list accompanies the article: *The dynamic landscape of peptide
function prediction* by Oriol Bárcenas, Carlos Pintado-Grima, Katarzyna
Sidorczuk, Felix Teufel, Henrik Nielsen, Salvador Ventura and Michał
Burdukiewicz (Oriol Bárcenas and Carlos Pintado-Grima are joint first
authors).

**Citation**: coming soon
